src/mpl/include/mpl_gpu.h

```diff
@@ -73,6 +73,7 @@ int MPL_gpu_query_support(MPL_gpu_type_t * type);
 int MPL_gpu_query_pointer_attr(const void *ptr, MPL_pointer_attr_t * attr);
 
 int MPL_gpu_ipc_handle_create(const void *ptr, MPL_gpu_ipc_mem_handle_t * ipc_handle);
+int MPL_gpu_ipc_handle_destroy(const void *ptr);
```

Comment:
Add comment that this is to be called in `ipc_handle_free_hook` and needed for fd-based IPC mechanism.

Reply:
Added in https://github.com/pmodels/mpich/pull/6478